makejdk.sh

```diff
@@ -129,6 +130,9 @@ parseCommandLineArgs()
       "--variant"  | "-bv" )
       export BUILD_VARIANT="$1"; shift;;
 
+      "--config-args"  | "-ca" )
```

Comment:
I would change this to `--configure-args` to make it consistent with the variable names and the name of the `configure` script